src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -99,10 +101,73 @@ public long LastRaftCommitEtag
 
         private Timer _timer;
         private RachisConsensus<ClusterStateMachine> _engine;
-        private bool _disposed;
-
+        private bool _disposed;      
         public RachisConsensus<ClusterStateMachine> Engine => _engine;
 
+        private ClusterMaintenanceMaster _clusterMaintenanceMaster;
+        public Dictionary<string, ClusterNodeStatusReport> ClusterStats()
+        {
+            if (_engine.LeaderTag != NodeTag)
+                return null; // not a leader
```

Comment:
It isn't a good idea to return null from a collection method, because that will likely turn into an NRE.
Either throw here or return an empty dictionary instead.